.github/workflows/add-hashicorp-contributed-label.yml

```diff
@@ -0,0 +1,28 @@
+name: Add HashiCorp contributed label
+
+# The purpose of this job is to label all HashiCorp contributed PRs, so that
+# we can more easily identify community contributed PRs (anything that doesn't
+# have this label).
+# While it might seem like this is the 'reverse' of what we should do, GitHub
+# (rightly) does not allow branches from forks to have write permissions, so
+# making PRs from forks self-label themselves as community-contributed is not
+# possible.
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
do we need the ready_for_review condition? I think the default conditions of opened, synchronize, reopened are sufficient

Reply:
Good point -- you're right. This runs on draft mode PRs so I think there's no reason to add ready_for_review.